pypesto/petab/importer.py

```diff
@@ -428,7 +428,10 @@ def create_objective(
 
         # create model
         if model is None:
-            model = self.create_model(force_compile=force_compile)
+            verbose = kwargs.pop('verbose', True)
```

Comment:
It's good to be able to pass `verbose=False`. However, it's potentially confusing to do that via `kwargs` (`**kwargs:
            Additional arguments passed on to the objective.`). I'd prefer adding (and documenting) it as a separate keyword argument to `create_objective`.

Reply:
True, will change.

Reply:
Additionally, I extended the same thing to all methods of the `PetabImporter` (create_edatas, create_solver etc...). To be uniform in changes, but it might be too much?

Reply:
> To be uniform in changes, but it might be too much?

No strong opinion. I don't think it will be used frequently via those other functions, but I don't think it hurts having it there.

Reply:
Same feeling